source/wp-content/themes/wporg-themes-2024/src/style/_listbox.scss

```diff
@@ -0,0 +1,70 @@
+/* Screenshot block styles */
+.wp-block-wporg-screenshot-preview {
+	border-color: var(--wp--preset--color--light-grey-1);
+}
+
+.wp-block-wporg-screenshot-preview:not(.is-preview-image):hover img {
```

Comment:
I find it a bit strange when hovering not to get a pointer style cursor, for elements I can interact with.

Reply:
Yeah, the cursor was raised in #128, so that'll be fixed.